scripts/testing/setup-mock-test-env.py

```diff
@@ -155,6 +159,15 @@ def parse_args():
     group.add_argument('--install', '-i', metavar='<packages>', action='store', type=str,
                        dest='install',
                        help="""install additional packages to the mock""")
+    group.add_argument('--install-pip', '-P', metavar='<pip packages>', action='store', type=str,
+                       dest='install_pip', default=None,
+                       help="""
+                       install additional packages from Python Package Index repository to the
+                       mock environment via the pip tool;
+                       there is a default set of pip packages which can be replaced by this
+                       parameter;
+                       set this command with empty string to disable pip completely
```

Comment:
This seems to be pretty hackish. When do we need to disable pip completely? It would be better to have an extra option for that, like --no-pip.

Reply:
I guess we can, should we also add the same for the rpm packages to be consistent?